tests/garage/tf/core/test_networks.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+import tensorflow as tf
+
+from garage.tf.core.networks import mlp
+from tests.fixtures import TfGraphTestCase
+
+
+class TestNetworks(TfGraphTestCase):
```

Comment:
Please add tests of batch normalization and weight normalization.

See missed lines here https://coveralls.io/builds/19684171/source?filename=garage%2Ftf%2Fcore%2Fnetworks.py#L21

Reply:
I discussed with Chang and we both agreed that batch normalization should be implemented later, since it is actually not being used at all, and it makes thing a little bit complicated. What do you think?